theme/templates/product.liquid

```diff
@@ -1,12 +1,14 @@
 {% assign current_variant = product.selected_or_first_available_variant %}
 {% assign featured_image = current_variant.featured_image | default: product.featured_image %}
-<img src="{{ featured_image | img_url: 'large' }}" alt="{{ featured_image.alt | escape }}" id="ProductPhotoImg">
+<div class="pt_625 site-padding-x bg-color-black">
+  <h1 class="uppercase serif-xl color-white">{{ product.title }}</h1>
+  <p class="uppercase sans-sm color-white">{{ product.vendor }}</p>
+  <img src="{{ featured_image | img_url: 'large' }}" alt="{{ featured_image.alt | escape }}" id="ProductPhotoImg">
 {% for image in product.images %}
```

Comment:
Below line 6 is everything else from the template that I'm leaving here until I work on the rest of the tickets for the Product page.